### PR TITLE
2022 edition

### PR DESCRIPTION
**Pull request by meyskens**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
+All talks will be presented online either pre-recorded or live (which one you prefer to give is up to you! We do however require a backup video to be uploaded).
 During the talks we will have an online Q&A and chatroom moderated by our friendly volunteers helping to provide a fun online experience.
 
 This is a great opportunity for new speakers, and we will have people helping you craft your talk and slides if so you wish. So if you, or someone you know, are working on something cool that might be good on a virtual stage, submit a proposal!
 
-Do you want to be one of our speakers? Please submit your proposal using [pentabarf](https://penta.fosdem.org/submission/FOSDEM21). The deadline for submission is January 2nd 2021 (23:59 UTC). If you are selected we would like to have the (backup) video uploaded by January 31st 2021.
+Do you want to be one of our speakers? Please submit your proposal using [pentabarf](https://penta.fosdem.org/submission/FOSDEM22). The deadline for submission is Decemnber 30th 2021 (23:59 UTC). If you are selected we would like to have the (backup) video uploaded by January 31st 2022.
```
Comment (meyskens) [inline README.md]:
```suggestion
Do you want to be one of our speakers? Please submit your proposal using [pentabarf](https://penta.fosdem.org/event/new). The deadline for submission is Decemnber 30th 2021 (23:59 UTC). If you are selected we would like to have the (backup) video uploaded by January 31st 2022.
```

File: README.md [hunk 2]
```diff
@@ -23,4 +23,3 @@ The room will follow Go’s [Code of Conduct](https://golang.org/conduct).
 
 Cheers,
 Maartje Eyskens
```
Comment (krisnova) [inline README.md:25]:
```suggestion
Maartje Eyskens
Kris Nóva
```

**Comment by krisnova** [comment]
If you could add me that would be great, otherwise LGTM

I think saying less about it being remote again is best for everyone. Hopefully next year we can all see each other again!

File: README.md [hunk 3]
```diff
@@ -2,15 +2,15 @@
...
+All talks will be presented online either pre-recorded or live (which one you prefer to give is up to you! We do however require a backup video to be uploaded).
 During the talks we will have an online Q&A and chatroom moderated by our friendly volunteers helping to provide a fun online experience.
 
 This is a great opportunity for new speakers, and we will have people helping you craft your talk and slides if so you wish. So if you, or someone you know, are working on something cool that might be good on a virtual stage, submit a proposal!
 
-Do you want to be one of our speakers? Please submit your proposal using [pentabarf](https://penta.fosdem.org/submission/FOSDEM21). The deadline for submission is January 2nd 2021 (23:59 UTC). If you are selected we would like to have the (backup) video uploaded by January 31st 2021.
+Do you want to be one of our speakers? Please submit your proposal using [pentabarf](https://penta.fosdem.org/event/new). The deadline for submission is Decemnber 30th 2021 (23:59 UTC). If you are selected we would like to have the (backup) video uploaded by January 31st 2022.
```
Comment (meyskens) [inline README.md]:
```suggestion
Do you want to be one of our speakers? Please submit your proposal using [pentabarf](https://penta.fosdem.org/event/new). The deadline for submission is Decemnber 30th 2021 (23:59 UTC). If you are selected we would like to have the (backup) video uploaded by January 20th 2022.
```

need editing time

**Pull request merged**